### DISCUSSION TITLE
Information on add-on products at check-in via PretixSCAN

### DISCUSSION THREAD

**Discussion by volkramweber** [pretix/pretix#3526] [Q&A]
I'm wondering how I can show information about add-on products for a ticket at check-in via PretixSCAN.
My idea was to use questions, but I couldn't figure out how to configure questions to be asked only if a ticket has a specific add-on. Is this possible?

The real life example: We sold festival tickets and some attendees bought car camp tickets as add-on. The entry crew should see this in PretixSCAN.

Is there a configurable solution?
If not, where/how could I start implementing it?

Thank you!

**Comment by raphaelm** [comment]
> Is there a configurable solution?

Currently not. We plan on adding it within the next few months, though.

> If not, where/how could I start implementing it?

This probably needs a little bit of a UI redesign of pretixSCAN across all supported platforms, so it's hard to give a specific pointer – we'll first need to figure that out ourselves.
